documentation/asciidoc/topics/yaml/xsite_expose_nodeport.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: NodePort
+          nodePort: 7800
```

Comment:
Node port has predefined range 30000-32767 by default
https://kubernetes.io/docs/concepts/services-networking/service/

Reply:
Thanks @dmvolod I've updated to mention the defaults.